### ISSUE TITLE
Client FPS 0, Server FPS 72. Unusable

### ISSUE THREAD

**Issue by crookedbag** [JackD83/ALVR#315]
I recently discovered ALVR and thought I'd try it out since I've got the oculus quest and my brother has Half-Life Alyx on steam so I decided to try and set it up on my new laptop (Also I'm not 100% sure this is the right place to post this so I'll post this on reddit too). After ALVR loads steamvr, I get about 48-52 fps for maybe 4 seconds and then the image freezes on my quest and the streaming statistics tell me there is 0 fps on client side?? This is confusing because server fps is 72. I'm new to this and unsure where to begin and there doesn't seem to be anything on here dealing with this kind of issue so I figured I'd ask.

I'm using a 5 Ghz router with standard of 802.11ax which should be the best right? I've also tried the mobile hotspot feature on my laptop at 5 Ghz band and still the same problem. I can't bring the router into my room so I was hoping this wasn't the issue. My GPU is GTX 1650 Ti and when steamvr is running, it goes upwards of 93% utilization (you can check out my task manager). If the GPU is the issue, I have an integrated radeon graphics card with my Ryzen 5 4600H chip so should I reinstall ALVR using that GPU? I've used the steamvr performance test on the 1650 ti GPU and says that its very capable and the games do run well without any problems on the PC's side through steamvr.

Here are images of what I'm seeing. Thanks for any help you can give, I really want to play Alyx!

![Screenshot 2020-10-13 025036](https://user-images.githubusercontent.com/72785347/95926639-c368b800-0d8a-11eb-8ddc-9dad8cef5dd2.png)
![task manager](https://user-images.githubusercontent.com/72785347/95926641-c4014e80-0d8a-11eb-9a4f-24f53bcd8ddb.png)

Also doing the steamvr performance test seems to be testing my integrated radeon graphics, but I have the 1650 Ti GPU selected as the default. Why is this?
![Steamvr](https://user-images.githubusercontent.com/72785347/95926640-c4014e80-0d8a-11eb-9848-b798afc66daf.png)

**Comment by The-Bob** [comment]
Can you try it with your laptop wired into your router? It looks like either a network or client issue to me. 

Also make sure that your headset and laptop (if you can't use it wired) are actually on the 5ghz band.

**Comment by crookedbag** [comment]
@The-Bob I was playing on the 12.3.0 release so I updated it to 12.4.1 but unfortunately that solved nothing. I checked the wifi settings on my computer and the quest and they both say the internet is at 5Ghz speed. The router is downstairs and I don't think my parents will let me take it to my room so I'm assuming I'm out of luck? Is there anything else that can be causing the client side to overload and freeze after 4 seconds? Thanks!
